middle_end/flambda/from_lambda/closure_conversion.ml

```diff
@@ -888,14 +890,10 @@ and close_one_function t ~external_env ~by_closure_id decl
   let code_id =
     Code_id.create ~name:(Closure_id.to_string closure_id) compilation_unit
   in
-  let unboxed_version =
-    Closure_id.wrap compilation_unit (Variable.create (
-      Ident.name (Function_decl.let_rec_ident decl)))
-  in
   let my_closure_id, is_curried =
     match Function_decl.kind decl with
     | Curried -> closure_id, true
-    | Tupled -> unboxed_version, false
+    | Tupled -> closure_id, false
```

Comment:
There is no need for binding my_closure_id like that anymore

Reply:
Fixed.